inst/extdata/emissions/EDGAR/EDGAR_CF4.csv

```diff
@@ -0,0 +1 @@
Non-Annex_I,Southeastern Asia,SGP,Singapore,2F7,Semiconductor/Electronics Manufacture,,5.34E-05,1.07E-04,2.62E-04,3.95E-04,5.05E-04,7.10E-04,7.64E-04,1.02E-03,1.37E-03,1.67E-03,1.73E-03,1.82E-03,2.30E-03,3.35E-03,2.79E-03,3.45E-03,4.27E-03,5.94E-03,6.37E-03,6.49E-03,6.60E-03,6.76E-03,8.10E-03,8.57E-03,1.04E-02,1.15E-02,1.31E-02,1.45E-02,1.91E-02,1.83E-02,3.22E-02,3.42E-02,3.67E-02,3.35E-02,3.40E-02,4.00E-02,4.24E-02,4.48E-02
Non-Annex_I,Indonesia +,IDN,Indonesia,2C,Production of metals,,,,,,,,,,,,,,6.95E-02,1.04E-01,1.09E-01,1.08E-01,1.06E-01,8.46E-02,9.06E-02,8.64E-02,7.35E-02,8.52E-02,7.58E-02,7.77E-02,7.18E-02,6.30E-02,5.29E-02,2.79E-02,1.86E-02,2.24E-02,1.89E-02,1.12E-02,1.40E-02,1.69E-02,1.76E-02,1.75E-02,1.69E-02,1.70E-02
Annex_I,Japan,JPN,Japan,2C,Production of metals,6.60E-01,8.04E-01,9.09E-01,9.90E-01,1.01E+00,9.18E-01,8.31E-01,1.07E+00,9.54E-01,9.09E-01,9.90E-01,6.99E-01,3.20E-01,2.33E-01,2.62E-01,2.08E-01,1.33E-01,4.75E-02,4.41E-02,4.56E-02,4.55E-02,4.69E-02,3.47E-02,3.47E-02,3.67E-02,4.18E-02,1.53E-02,1.53E-02,1.44E-02,9.90E-03,5.88E-03,5.97E-03,5.77E-03,5.85E-03,5.40E-03,6.30E-03,6.30E-03,6.30E-03,6.30E-03
Annex_I,Japan,JPN,Japan,2F7,Semiconductor/Electronics Manufacture,2.14E-05,1.16E-03,2.29E-03,5.70E-03,8.54E-03,1.09E-02,1.52E-02,1.63E-02,2.17E-02,2.93E-02,3.58E-02,3.69E-02,3.88E-02,4.92E-02,7.18E-02,5.97E-02,7.40E-02,9.16E-02,1.27E-01,1.36E-01,1.39E-01,1.39E-01,1.39E-01,1.73E-01,1.91E-01,2.34E-01,2.48E-01,2.61E-01,2.75E-01,2.90E-01,3.04E-01,2.92E-01,2.82E-01,2.70E-01,2.66E-01,2.63E-01,2.50E-01,2.41E-01,2.32E-01
Annex_I,Oceania,AUS,Australia,2C,Production of metals,1.76E-01,2.28E-01,2.22E-01,2.18E-01,2.19E-01,2.34E-01,2.31E-01,2.48E-01,2.73E-01,2.53E-01,2.20E-01,2.68E-01,2.22E-01,1.99E-01,3.05E-01,3.95E-01,3.89E-01,4.07E-01,4.41E-01,5.06E-01,5.01E-01,4.52E-01,3.86E-01,2.91E-01,2.43E-01,2.08E-01,2.00E-01,1.84E-01,1.78E-01,1.64E-01,1.47E-01,1.24E-01,1.01E-01,1.02E-01,1.04E-01,1.04E-01,1.06E-01,1.06E-01,1.04E-01
Annex_I,Oceania,AUS,Australia,2F7,Semiconductor/Electronics Manufacture,,2.00E-06,2.00E-06,1.07E-05,1.87E-05,2.67E-05,3.47E-05,3.47E-05,4.27E-05,5.94E-05,6.94E-05,6.94E-05,7.14E-05,9.01E-05,1.28E-04,1.09E-04,1.36E-04,1.70E-04,2.40E-04,2.59E-04,2.62E-04,2.64E-04,2.66E-04,3.29E-04,3.62E-04,5.22E-04,6.52E-04,6.18E-04,7.87E-04,5.83E-04,4.09E-04,6.04E-04,7.46E-04,7.92E-04,8.75E-04,8.32E-04,1.01E-03,1.09E-03,1.18E-03
Annex_I,Oceania,NZL,New Zealand,2C,Production of metals,,1.10E-02,4.40E-02,5.80E-02,5.50E-02,5.45E-02,6.90E-02,7.20E-02,7.50E-02,7.70E-02,7.75E-02,7.70E-02,8.15E-02,1.10E-01,1.22E-01,1.21E-01,1.16E-01,1.21E-01,1.20E-01,1.18E-01,1.17E-01,1.10E-01,9.72E-02,1.04E-01,7.49E-02,8.60E-02,7.92E-02,6.80E-02,6.01E-02,5.18E-02,4.20E-02,3.14E-02,2.24E-02,2.27E-02,2.34E-02,2.35E-02,2.25E-02,2.36E-02,2.11E-02
```

Comment:
Wait, the EDGAR files are already in the repository! Why are you adding them again?